From 60054686e2c1a4bedf1d507af97ebbb7ff491e77 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Thu, 18 Jul 2019 15:23:15 +0800
Subject: [PATCH] rt_tgsigqueueinfo: disable test on musl

Fix build error with musl:
rt_tgsigqueueinfo01.c: In function 'sigusr1_handler':
rt_tgsigqueueinfo01.c:42:22: error: 'siginfo_t' {aka 'struct <anonymous>'} has no member named '_sifields'; did you mean '__si_fields'?
   42 |  sigval_rcv = uinfo->_sifields._rt.SI_SIGVAL.sival_ptr;
      |                      ^~~~~~~~~
      |                      __si_fields

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 testcases/kernel/syscalls/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/kernel/syscalls/Makefile b/testcases/kernel/syscalls/Makefile
index d1becd0..1f3ff34 100644
--- a/testcases/kernel/syscalls/Makefile
+++ b/testcases/kernel/syscalls/Makefile
@@ -31,7 +31,7 @@ endif
 ifeq ($(LIBC),musl)
 FILTER_OUT_DIRS	+= confstr fmtmsg getcontext ioctl mallopt profil \
 		   rt_sigsuspend setdomainname sethostname sigsuspend \
-		   ustat
+		   ustat rt_tgsigqueueinfo
 endif
 
 ifeq ($(UCLIBC),1)
-- 
2.7.4

